utils/compare_ds.py

```diff
@@ -51,381 +31,52 @@ def parse_args():
         "--only-rules", action="store_true",
         help="Print only removals from rule set."
     )
+    parser.add_argument(
+        "--rule-diffs", action="store_true",
+        help="Output diffs per rule, instead of a single diff. "
+             "The rule diffs are outptu to directory './compare_ds-diffs/'."
+    )
+    parser.add_argument(
+        "--output-dir", metavar="OUTPUT_DIR",
+        type=str, action="store", default="./compare_ds-diffs",
+        help="Directory where rule diff files will be saved. Only used with --rule-diffs option. "
+             "If the directory doesn't exist, it will be created."
+    )
+    parser.add_argument(
+        "--disa-content", action="store_true",
+        help="This option enables comparison betwween DISA contents by ignoring the release "
+             "number in rule IDs."
+    )
```

Comment:
```suggestion
    parser.add_argument(
        "--rule-diffs", action="store_true",
        help="Output diffs per rule, instead of a single diff. "
             "The rule diffs are output to directory './compare_ds-diffs/'."
    )
    parser.add_argument(
        "--output-dir", metavar="OUTPUT_DIR",
        type=str, action="store", default="./compare_ds-diffs",
        help="Directory where rule diff files will be saved. Only used with --rule-diffs option. "
             "If the directory doesn't exist, it will be created."
    )
    parser.add_argument(
        "--disa-content", action="store_true",
        help="This option enables comparison between DISA contents by ignoring the release "
             "number in rule IDs."
    )
```